src/query/storages/common/table-meta/src/meta/v5/snapshot.rs

```diff
@@ -103,6 +102,16 @@ impl TableSnapshot {
         // trim timestamp to micro seconds
         let trimmed_timestamp = trim_timestamp_to_micro_second(adjusted_timestamp);
         let timestamp = Some(trimmed_timestamp);
+        let prev_snapshot_id = match prev_id {
+            Some((prev_snapshot_id, prev_format)) => {
+                if prev_format <= 4 {
+                    Some((None, prev_snapshot_id, prev_format))
```

Comment:
If we read an old snapshot, we think the timestamp is very old, and we will vacuum it? How to process the default timestamp, or read it from snapshot,right?